### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/dalton/actions/runs/3814722224#summary-10379588841

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/dalton/actions/runs/3814722360#summary-10379588987

I've included changes to `app/static/engine-configs`, but they can of course be dropped.

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -392,7 +392,7 @@ number of user-configurable options:
         "Alert Details" tab on the job results page. Information returned
         includes hex/ASCII output from packets that generated alerts as
         well as "Extra" data from the unified2 file such as "Original
-        Client IP" from packets with "X-Forwared-For" or "True-Client-IP"
+        Client IP" from packets with "X-Forwarded-For" or "True-Client-IP"
```
Comment (jsoref) [inline README.rst:395]: there are some famously misspelled items in http headers, but I don't think this is amongst them

File: README.rst [hunk 2]
```diff
@@ -570,12 +570,12 @@ page or directly downloaded using the following URL::
 Sensors
 =======
 
-Agents (a.k.a. "Sensors") check in to the Dalton server frequently
+Agents (a.k.a. "Sensors") check into the Dalton server frequently
```
Comment (jsoref) [inline README.rst:573]: correcting `in to` to `into` has been fairly controversial, but I believe this is correct.

File: app/dalton.py [hunk 3]
```diff
@@ -920,7 +920,7 @@ def page_coverage_jid(jid, error=None):
                     if tech not in sensors:
                         sensors.append(tech)
             except Exception as e:
-                return render_template('/dalton/error.hml', jid=None, msg="Error getting sensor list for %s.  Error:\n%s" % (tech, e))
+                return render_template('/dalton/error.html', jid=None, msg="Error getting sensor list for %s.  Error:\n%s" % (tech, e))
```
Comment (jsoref) [inline app/dalton.py:923]: i'm pretty sure this is a significant bugfix

File: app/dalton.py [hunk 4]
```diff
@@ -1416,7 +1416,7 @@ def page_coverage_summary():
         if request.form.get('optionProdRuleset') and request.form.get('optionEnableAllRules'):
             bEnableAllRules = True
 
-        # get showFlobitAlerts option
+        # get showFlowbitAlerts option
```
Comment (jsoref) [inline app/dalton.py:1419]: brand?

File: dalton-agent/dalton-agent.conf [hunk 5]
```diff
@@ -62,13 +62,13 @@ KEEP_JOB_FILES = False
...
 # This will attempt to support Suricata Socket Control
-# mode  but will get disabled automatically if the sesnor doesn't support it
+# mode  but will get disabled automatically if the sensor doesn't support it
 # and ultimately, the use of Suricata Socket Control (for supported sensors) is
 # determined by a parameter from the passed in job. 
 USE_SURICATA_SOCKET_CONTROL = True
 
 # Location of Suricata Socket Control Python Module (should be
-# included with Suriata source), if not in PYTHONPATH. These are
+# included with Suricata source), if not in PYTHONPATH. These are
```
Comment (jsoref) [inline dalton-agent/dalton-agent.conf:71]: brand

**Review by jsoref — COMMENTED** [review]
A bunch of these corrections were suggested by Google Sheets, but I did quite a few one-off fixes on my own (way more than normal, I'm actually disappointed in Google Sheets).

All fault mine.

**Comment by whartond** [comment]
Note that the files in `engine-configs` are taken straight from the source (e.g. Suricata repo), with minimal modifications (if any).  Spelling errors in those should be reported upstream (although many of them have been addressed already) and won't be bothered with here.

**Comment by jsoref** [comment]
Dropped. Fwiw, I opened https://github.com/OISF/suricata/pull/7708 to them a while ago...

**Pull request merged**